.eslintrc.js

```diff
@@ -66,9 +66,6 @@ module.exports = {
       "peerDependencies": false
     }],
     "func-call-spacing": "off",
-    "max-len": ["error", {
```

Comment:
is this meant to be here?

Reply:
Rule conflicts with prettier on import statement